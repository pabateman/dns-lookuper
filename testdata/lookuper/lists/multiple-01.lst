terraform.io hashicorp.com
invalid$name
