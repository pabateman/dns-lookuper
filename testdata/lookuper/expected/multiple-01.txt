139.178.84.217
192.0.43.8
76.76.21.21
